edx_solutions_api_integration/users/serializers.py

```diff
@@ -87,7 +87,7 @@ def get_profile_image(self, user):
 
     def get_courses_enrolled(self, user):
         """ Serialize user enrolled courses """
-        enrollments = user.courseenrollment_set.all()
+        enrollments = user.courseenrollment_set.filter(is_active=1)
```

Comment:
Since we only need course_id, following would be a bit more efficient:
`user.courseenrollment_set.filter(is_active=1).values_list('course_id', flat=True)`

Reply:
Addressed